lib/osf-components/addon/components/funding-metadata/component.ts

```diff
@@ -44,10 +45,27 @@ export default class FundingMetadata extends Component<Args> {
     @action
     validateFunderObjects() {
         for (const item of this.funderObjects) {
-            if (item.award_number || item.award_title || item.award_uri || item.funder_name) {
+            if (!(item.award_number || item.award_title || item.award_uri)) {
+                if (!item.errors.includes(this.intl.t('osf-components.funding-metadata.error'))) {
+                    item.errors.pushObject(this.intl.t('osf-components.funding-metadata.error'));
+                }
+            } else {
                 item.errors.removeObject(this.intl.t('osf-components.funding-metadata.error'));
-            } else if (item.errors.length === 0) {
-                item.errors.pushObject(this.intl.t('osf-components.funding-metadata.error'));
+            }
```

Comment:
I don't think we need this any more. We just needed one of those four items including funder_name, and now funder_name is always required, so this secondary condition shouldn't be needed anymore.